### DISCUSSION TITLE
how to hide mobile scan function?

### DISCUSSION THREAD

**Discussion by Bot80926** [WalletConnect/walletconnect-monorepo#2667] [Developer Support]
My boss think the QR code for mobile is too large to confuse users.
Can i hide the mobile connect qrCode? 
Or can i change the QR code css styles to make QR code smaller size ?
<img width="366" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/127970265/7e037245-0c5a-4b2c-babc-8ba26289fe5e">

**Comment by xzilja** [comment]
Qrcode has a lot of data, so making it smaller will likely make it unusable. There is no option to hide it, but you can always override shadow dom css to do what you want i.e. hide it
